src/test/java/io/tiledb/java/api/QueryConditionTest.java

```diff
@@ -55,14 +55,14 @@ public void setup() throws Exception {
   public void arrayCreate() throws Exception {
     // Create getDimensions
     Dimension<Long> d1 =
-        new Dimension<Long>(ctx, "d1", Long.class, new Pair<Long, Long>(1l, 3l), 2l);
-    Dimension<Long> d2 =
-        new Dimension<Long>(ctx, "d2", Long.class, new Pair<Long, Long>(1l, 3l), 2l);
+        new Dimension<Long>(ctx, "d1", Long.class, new Pair<Long, Long>(1l, 9l), 2l);
+    //    Dimension<Long> d2 =
+    //        new Dimension<Long>(ctx, "d2", Long.class, new Pair<Long, Long>(1l, 3l), 2l);
 
     // Create and set getDomain
     Domain domain = new Domain(ctx);
     domain.addDimension(d1);
-    domain.addDimension(d2);
+    //    domain.addDimension(d2);
```

Comment:
why removed, here and above?

Reply:
I was testing something and forgot to revert to the original. Should be ok now.